tests/yaml_configs/abi_test_low_memory.yaml

```diff
@@ -6,18 +6,18 @@ resampled_read: True
 available_sectors:
   static:
     sector_list:
-        - goes16
+      - goes_east
```

Comment:
You must have a YAML formatter in vscode also! I'm going to open an issue on VSCode repository to finalize all the settings for yaml and toml and rst auto-formatting.